godep/plan.sh

```diff
@@ -0,0 +1,11 @@
+pkg_name=godep
```

Comment:
I'm with Elliott on this one but, I hope there are no other `dep` commands out there for other languages since it is very generic. 😅